source/manual1/index.rst

```diff
@@ -3,4 +3,4 @@
    :maxdepth: 1
    :caption: Contents
 
-   PartA
\ No newline at end of file
+   ../content/PartA
```

Comment:
Maybe I am wrong on this change, but otherwise, I was not able to link the corresponding documents as intended. Please advise.

Reply:
True. I don't know what I did there.